src/react/src/util/styles.js

```diff
@@ -292,3 +293,308 @@ export const makeCustomDropdownIndicatorStyles = Object.freeze({
         marginRight: '0.5em',
     }),
 });
+
+export const makeContributeProductionLocationStyles = theme =>
```

Comment:
Pls, make sure that all style objects below won't require major refactoring to be ported on mobile.

Reply:
I tried to split the components into logical blocks to make it possible to add breakpoints for different devices. 
However, it's difficult to predict everything because there are many unknown factors.

Reply:
Ack.